### ISSUE TITLE
WiiSXR - 2 Issues

### ISSUE THREAD

**Issue by Restart80** [Mystro256/wiisxr#14]
Alright, I have recently started playing WiiSX/WiiSXR (now SXR) and have two main issues:

-In Digimon World (and probably other games?), my controller randomly becomes unresponsive for a couple of seconds. If I was walking in a certain direction, it keeps moving until I regain control. This happens every minute or so for no apparent reason. I use a Gamecube controller, as with anything else.

-WiiSX (and probably SXR?) claims that it has native save support for Gamecube Memory Cards. I have tried to use physical Gamecube Memory Cards to save, but they do not. The emulator simply saves to my USB every time. I've tried selecting "Slot A" and "Slot B" as the native saves device, yet I can take out my memory cards and it will load the saves just fine because it is saving to the USB (confirmed since I check the save folder on my computer as well). This is somewhat of a problem because it is hard, if not impossible to purposefully save to slot 2 in some games. In Digimon Digital Card Battle, the game does NOT give you a choice on which slot to save; it is slot 1 by default. This makes it impossible to do 2-player battles because I cannot manipulate the slots in for USB in the same way I could manipulate physical memory cards, if you get what I'm saying.

Otherwise, I'm pretty impressed with the fact that I am playing 50-60 FPS PS1 games on my Wii.

**Comment by Mystro256** [comment]
Some comments...

Controler:
- Do you only experience this with a Gamecube controller? or can you reproduce this with another setup? Was this an issue with WiiSX too?

Card:
- Unfortunately, I only own a Wii U, so I can't really debug this. Hopefully someone else has interest in fixing these and can look into these issues.

**Comment by Restart80** [comment]
Controller:

I tried using a Wii Remote, but there is still the same problem. This has been an issue for both WiiSX and WiiSXR.

Card:

Also when I click "Save Memcard" when Slot A or B is selected, even when I make sure the USB isn't selected as the save device, the emulator will actually freeze.

**Comment by MalphasX** [comment]
Can confirm the same thing here. I was playing Herc's Adventures on WiiSX and then tried WiiSXR and both have the controller just freeze for a little bit. Was with a Gamecube controller for me. Also happy with the 60 fps. Sadly the game has some issues still, but maybe someday the game will be perfectly playable. I could probably already play it with savestates if the controller thing is fixed.

**Comment by Restart80** [comment]
Yeah, was hoping this would be fixed in 2.2 but it's still there. My games run almost perfect except for that intermittent controller freezing.

Memory Card thing is no longer an issue for me since I figured out how to save to the USB stick properly.
